### ISSUE TITLE
Unrestricted Dungeon Pass

### ISSUE THREAD

**Issue by YaibaToKen** [aura-project/aura#557]
Just to let everyone know that it's currently impossible to use the Unrestricted Dungeon Passes as your character will freeze. 

[Unimplemented] - PacketHandlerManager: Handler for 'A950', '?'.

This is the message logged by the Channel Server

**Comment by xCryptic** [comment]
That makes sense since it hasn't been implemented. No need for an issue since it's not a g1/g2 item.

**Comment by YaibaToKen** [comment]
Was meant as more of a warning since the player pretty much has to restart the client or server losing all progress for that login

**Comment by eotetb** [comment]
I'd recommend not making that item available to players until those passes are implemented. By default those are unobtainable.

**Comment by exectails** [comment]
Pretty much what @eotetb said, don't let players have items that aren't implemented yet^^ I appreciate the warning for others, but there's *so many* items we'd have to warn about, creating a list or something isn't really viable. It would be easier to make a list of items that *are* actually working^,^

Basically, most (semi-)usable items that aren't obtainable during normal game play aren't working yet.

**Comment by YaibaToKen** [comment]
Yeah xD I got the pass from the gachapons I generated based on the wiki

**Comment by exectails** [comment]
I see^^ I've noticed this to be a general problem when adding gachapons, because you get to items that don't work yet, or shouldn't be accessible in G1, very quickly. In the gachapon(-like) items that I added and made available through events and stuff already I commented or left out those items for that reason.

I'm thinking gachapons are, in a way, very specific to the server they're accessible on. We can recreate the NA gachapons, but in reality those were created for NA, for the time when they were released, and in some cases other regions actually had completely different ones. I decided to see them more dynamically than other features, something where the functionality partially has to be decided by the user.
